[BUG] App launch fail with the error - Can't Resolve './App.css'

**Describe the bug**
After starting the front-end server, the application crashes as soon as the launch completes and the following error is displayed:

```
ERROR in ./src/App.jsx 7:0-19
Module not found: Error: Can't resolve './App.css' in 'E:\Open Source\os-Farmhub\frontend\src'
```

**To Reproduce**
Steps to reproduce the behavior:
1. Run ```npm install```
2. Run ```npm start```
3. See error

**Expected behavior**
The homepage should be visible on launch.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop:**
 - OS: Windows 11 Home
 - Browser:  Google Chrome
 - Version: 117:0:5938:149

--- a/frontend/src/App.css
+++ b/frontend/src/App.css
@@ -1,0 +1,1 @@
+/* Define App level styling here */

